src/CommunityToolkit/Xamarin.CommunityToolkit/PlatformConfiguration/Multiplatform/StatusBar/BarStyle.android.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using Android.OS;
+using Android.Views;
+using Xamarin.Forms.Platform.Android;
+using Debug = System.Diagnostics.Debug;
+
+namespace Xamarin.CommunityToolkit.PlatformConfiguration.Multiplatform
+{
+	static class BarStyle
+	{
+		internal static bool IsSupported()
+		{
+			if (Build.VERSION.SdkInt < BuildVersionCodes.M)
+			{
+				Debug.WriteLine($"This functionality is not available. Minimum supported API is {BuildVersionCodes.M}");
```

Comment:
Don't you mind removing this debug printing?

If I understand correctly, it won't be visible for devs-SDK-consumers anyway, cuz we deliver "release" build via nuget. So, all Debug writes will be ignored.

```internal static bool IsSupported() => Build.VERSION.SdkInt >= BuildVersionCodes.M;``` is sufficent from my perspective

Reply:
@pictos [asked](https://github.com/xamarin/XamarinCommunityToolkit/pull/812#discussion_r603644830) to add some logging when this functionality is not supported. Developer would want to know why they changed statusbar color, and nothing happened.

I can use `Trace` or Xamarin's `Log` instead. But they are not yet used anywhere in XCT.

Reply:
I would say that the Xamarin's log is a better option

Reply:
@pictos I'm a little concerned about why it's in `Xamarin.Forms.Internals` namespace. Feels like things in `Internals` namespace are not intended to be used outside XF assemblies.

On the other hand, why it's public then?

Reply:
For this situation I think that is ok to use it

Reply:
Replaced `Debug` with `Log`.
@AndreiMisiukevich Are you ok with this approach?

Reply:
@maxkoshevoi perfect!